_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -4821,11 +4821,13 @@
 "ajE" = (
 /obj/machinery/atmospherics/components/unary/vent_pump/on,
 /obj/effect/landmark/secequipment,
+/obj/effect/turf_decal/bot,
```

Comment:
Meta's security division doesn't have decals under any of the other lockers (sans the bomb & biosuit ones in the armory). This seems really out of place to me.